File: plusmall-business-web/src/main/webapp/js/service/typeTemplateService.js
app.service('typeTemplateService',function ($http) {

    //根据Id查找单个模板数据
    this.findOne=function (id) {
        return $http.get('../typeTemplate/findOne.do?id='+id);
    }

    //查询规格列表
    this.findSpecList=function (id) {
        return $http.get('../typeTemplate/findSpecList.do?id='+id);
    }
})

File: plusmall-business-interface/src/main/java/com/plusmall/business/TypeTemplateService.java
package com.plusmall.business;

import com.plusmall.model.TbTypeTemplate;

import java.util.List;
import java.util.Map;

/**
 * @Description:
 */
public interface TypeTemplateService {
	public TbTypeTemplate findOne(Long id);
	public List<Map> findSpecList(Long id);
}
